### PR TITLE
Regularize ammunition naming

### PR DESCRIPTION
**Pull request by Muffindrake**
#### Summary
SUMMARY: Interface "Regularize ammunition naming"

#### Purpose of change
In an alphanumeric listing of items, the location of ammunition of the same type is all over the place, and very often not even on the same screen if there happen to be non-ammunition items among them. The ammunition named `reloaded` is a particularly bad offender in this, the unfortunate choice of the letter `r` ensuring it ends up at the end of multi-page item listings, ludicrously far away from the ammo that usually starts with a period.

#### Describe the solution
Change all reloaded and blackpowder variants of ammunition to use a scheme `type (comma-separated list of modifiers)`, e.g. `00 shot (reloaded, blackpowder)`.

#### Describe alternatives you've considered
Different naming schemes: 
`type, comma separated list of modifiers`

`type (modifier1) (modifier2)`

#### Additional context
before
after

### CODE REVIEW

**Comment by ZhilkinSerg** [comment]
Poor translators xD

File: data/json/items/ammo/10mm.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "id": "bp_10mm_fmj",
     "copy-from": "10mm_fmj",
     "type": "AMMO",
-    "name": "10mm Auto FMJ blackpowder",
+    "name": "10mm Auto FMJ (reloaded, blackpowder)",
```
Comment (cainiaowu) [inline data/json/items/ammo/10mm.json]: IMHO "blackpowder" already suggests the item is "reloaded".

**Comment by DaviBones** [comment]
Parentheses in item names typically indicate transient or reversible properties -- `(frozen)`, `(wet)`, `(hot)`, and `(padded)`, to name a few. I suggest doing
```
00 shot, reloaded
00 shot, blackpowder
```
for the sake of consistency.

Also, to add on to cainiaowu's point of redundancy, there are two further layers of redundancy:
- The fact that blackpowder ammo is reloaded is stated in the item description
- Blackpowder ammo doesn't spawn afaik, so the player will always have crafted it, meaning they know quite well that it is reloaded.

File: data/json/items/ammo/10mm.json (hunk 1 above)
Comment (DaviBones) [reply data/json/items/ammo/10mm.json]: To add to this, there are two further layers of redundancy:
- The fact that blackpowder ammo is reloaded is stated in its item description.
- Blackpowder ammo doesn't spawn afaik, so the player will always have crafted it, meaning they know quite well that it is reloaded.

**Comment by esotericist** [comment]
> > Parentheses in item names typically indicate transient or reversible properties -- `(frozen)`, `(wet)`, `(hot)`, and `(padded)`, to name a few. I suggest doing
> > ```
> > 00 shot, reloaded
> > 00 shot, blackpowder
> > ```
> > 
> > 
> > for the sake of consistency.
> 
> I fully agree. I had done it the current way because some dev in the dev discord had some trepidations about the commas for some reason. I much preferred doing this type of scheme to begin with.

Bit belated, but do you remember any other details about which dev had which concern about the commas? I'm not able to find anything about that.

File: data/json/items/ammo/4570.json [hunk 2]
```diff
@@ -78,7 +78,7 @@
     "//": "***Add blackpowder effects once #30944 gets merged***",
     "copy-from": "4570_low",
     "type": "AMMO",
-    "name": "reloaded .45-70 black powder",
+    "name": ".45-70, blackpowder",
```
Comment (ymber) [inline data/json/items/ammo/4570.json]: Missing reloaded.

Comment (Muffindrake) [reply data/json/items/ammo/4570.json]: It was established that `blackpowder` already implies it's reloaded, so I don't see a reason to add that here (the original name was wrong to this end, anyway).

**Comment by Muffindrake** [comment]
There is no actual word `blackpowder` in any English dictionary I know of, so I'm changing all occurrences to `black powder` as it should be.

**Pull request merged**